tests/config/test_data.config

```diff
@@ -211,14 +211,8 @@ params {
                 test2_genome_vcf_gz_tbi                        = "${test_data_dir}/genomics/homo_sapiens/illumina/gvcf/test2.genome.vcf.gz.tbi"
                 test2_genome_vcf_idx                           = "${test_data_dir}/genomics/homo_sapiens/illumina/gvcf/test2.genome.vcf.idx"
 
-                test_broadpeak                                 = "${test_data_dir}/genomics/homo_sapiens/illumina/broadpeak/test.broadPeak"
-                test2_broadpeak                                = "${test_data_dir}/genomics/homo_sapiens/illumina/broadpeak/test2.broadPeak"
-
-                test_narrowpeak                                = "${test_data_dir}/genomics/homo_sapiens/illumina/narrowpeak/test.narrowPeak"
-                test2_narrowpeak                               = "${test_data_dir}/genomics/homo_sapiens/illumina/narrowpeak/test2.narrowPeak"
-
-                test_10x_1_fastq_gz                           = "${test_data_dir}/genomics/homo_sapiens/illumina/10xgenomics/test.10x_1.fastq.gz"
-                test_10x_2_fastq_gz                           = "${test_data_dir}/genomics/homo_sapiens/illumina/10xgenomics/test.10x_2.fastq.gz"
+                test_10x_1_fastq_gz                           = "${test_data_dir}/genomics/homo_sapiens/illumina/10xgenomics/test_10x_S1_L001_R1_001.fastq.gz"
+                test_10x_2_fastq_gz                           = "${test_data_dir}/genomics/homo_sapiens/illumina/10xgenomics/test_10x_S1_L001_R2_001.fastq.gz"
```

Comment:
Can we rename these files on nf-core/testdatasets to their original names i.e. `test.10x_1.fastq.gz` and `test.10x_2.fastq.gz`

Reply:
Negative. Cell ranger only supports Illumina file naming for the moment, sadly. :disappointed: